src/App/Options/Cmd.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE DeriveAnyClass   #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TemplateHaskell  #-}
+
+module App.Options.Cmd where
+
+import App.Options.Cmd.Help       as C
+import App.Options.Cmd.KafkaToSqs as C
+import App.Options.Cmd.SqsToKafka as C
+import Control.Lens
+import Data.Monoid
+import Options.Applicative
+
+data Cmd
+  = CmdOfCmdHelp        { _cmdHelp          :: CmdHelp        }
+  | CmdOfCmdKafkaToSqs  { _cmdKafkaToSqs    :: CmdKafkaToSqs  }
+  | CmdOfCmdSqsToKafka  { _cmdCmdSqsToKafka :: CmdSqsToKafka  }
```

Comment:
Is `sqs-to-kafka` also included in this repo? then it needs a better name not indicating just kafka to sqs.

Reply:
Yeah, should rename the project.

Reply:
`kafka-bind` maybe?

Reply:
maybe :) 

then we get the new sqs connector for free :)